virttest/env_process.py

```diff
@@ -1174,7 +1175,10 @@ def preprocess(test, params, env):
     kvm_module_params = params.get("kvm_module_parameters", "")
     force_load = params.get("kvm_module_force_load", "no") == "yes"
     global KVM_MODULE_HANDLER
-    KVM_MODULE_HANDLER = utils_kernel_module.reload("kvm", force_load, kvm_module_params)
+    # Only unload/reload the kvm_probe module is required
+    kvm_module = arch.get_kvm_module_list()[-1].replace('-', '_')
```

Comment:
Instead of replacing with the arch specified module directly, it's better to introduce another routine to handle the modification of arch modules.